packages/demos-babylon/src/customCharacterDemo.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Custom Character Demo</title>
+
+    <link rel="stylesheet" href="customCharacterDemo.css" />
+
+    <!-- Babylon.js dependencies -->
+    <script src="https://cdn.jsdelivr.net/npm/babylonjs@4.2.1/babylon.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/babylonjs-loaders@4.2.1/babylonjs.loaders.min.js"></script>
```

Comment:
Should these be bundled with webpack? I also see a testing risk, an engineer could upgrade the version of babylon in package.json, try out the sample and think 'upgrade went well', but they didn't actually test with the new version in this sample

Reply:
I think your suggestion is good. Let's consider it out of scope for this PR, but I'm still working on demos and will incorporate that change into another PR. Good?